proofs/cbmc/poly_uniform_4x/poly_uniform_4x_harness.c

```diff
@@ -6,7 +6,10 @@
 void harness(void)
 {
   poly *r0;
-  uint8_t(*seed)[MLD_ALIGN_UP(MLDSA_SEEDBYTES + 2)];
+  poly *r1;
+  poly *r2;
+  poly *r3;
+  uint8_t seed[4][MLD_ALIGN_UP(MLDSA_SEEDBYTES + 2)];
```

Comment:
Can you change the type of seed back please?

Reply:
Like this:
```
uint8_t(*seed)[MLD_ALIGN_UP(MLDSA_SEEDBYTES + 2)];
```
my clang-format kept changing it to:
```
uint8_t (*seed)[MLD_ALIGN_UP(MLDSA_SEEDBYTES + 2)];
```
(adding 1 space), then it failed the "lint" check.

I will change it back, though.

Reply:
Hmm... inside the NIX shell, it says 19.1.7.

Outside NIX (and therefore in my EMACS buffer), it says 20.1.5